modules/claims_api/app/models/claims_api/auto_established_claim.rb

```diff
@@ -370,6 +370,9 @@ def transform_service_branch
 
       transformed_service_periods = received_service_periods.map do |period|
         name = period['serviceBranch']
+
```

Comment:
Kayla asked that we provide a simple log that'll show what `serviceBranch` values we're seeing.  Then maybe we can get some insight into what values consumers are using more often/not at all.